src/commands.py

```diff
@@ -358,39 +356,39 @@ def add_tax_bracket(
         start: Fraction, end: Fraction,
         rate: Fraction, name: str, server: Server):
     """Add a tax bracket to a server with authorization from author"""
-    author = _get_account(author, server)
-    _assert_authorized(author, None)
+    author_acc = get_account(author, server)
+    _assert_authorized(author_acc, None)
     server.add_tax_bracket(author, start, end, rate, name)
 
 
 def remove_tax_bracket(
         author: Union[AccountId, str],
         name: str, server: Server):
     """Remove tax bracket by name with authorization from author"""
-    author = _get_account(author, server)
-    _assert_authorized(author, None)
+    author_acc = get_account(author, server)
+    _assert_authorized(author_acc, None)
     server.remove_tax_bracket(author, name)
 
 
 def force_tax(author: Union[AccountId, str], server: Server):
     """Manually trigger taxation"""
-    author = _get_account(author, server)
-    _assert_authorized(author, None)
+    author_acc = get_account(author, server)
+    _assert_authorized(author_acc, None)
     server.force_tax(author)
 
 
 def auto_tax(author: Union[AccountId, str], server: Server) -> bool:
     """Toggle automatic taxation"""
-    author = _get_account(author, server)
```

Comment:
all of this will literally fail
why change it from what it was if the new one actually fails?
maybe better to just leave it